common/src/test/java/org/apache/sedona/common/raster/FunctionsTest.java

```diff
@@ -76,6 +77,14 @@ public void valueWithMultibandRaster() throws TransformException {
         assertEquals(255d, PixelFunctions.value(multiBandRaster, point(4.5d,4.5d), 4), 0.1d);
     }
 
+    @Test
+    public void testPixelAsPolygon() throws FactoryException, TransformException {
+        GridCoverage2D emptyRaster = RasterConstructors.makeEmptyRaster(1, 5, 10, 123, -230, 8);
```

Comment:
Please add tests testing skewed rasters as well. Make sure you cross check results using PostGIS